lib/run.js

```diff
@@ -87,6 +99,13 @@ function run (opts, cb) {
   url.setupClient = opts.setupClient
   url.timeout = opts.timeout
   url.requests = opts.requests
+  url.responseMax = opts.maxConnectionRequests
+
+  if (!url.responseMax) {
+    url.responseMax = typeof opts.maxOverallRequests === 'number'
+                      ? (opts.maxOverallRequests / opts.connections) : false
+    // this needs work if ((maxOverallRequests % opts.connections) > 0)
+  }
```

Comment:
I don't like that comment over `maxOverallRequests % opts.connections > 0`.  Sort it out, or `cb(new Error('maxOverallRequest must be multiple of the number of connections')`.